src/v/resource_mgmt/storage.cc

```diff
@@ -312,7 +303,7 @@ eviction_policy::collect_reclaimable_offsets() {
             .handle_exception_type([](const ss::gate_closed_exception&) {})
             .handle_exception([ntp = p->ntp()](std::exception_ptr e) {
                 vlog(
-                  rlog.debug,
```

Comment:
interesting, wonder why you didn't bump it to warn?

Reply:
I waffled on this and originally was just erring on the side of caution in case this actually ends up being logged a lot and being scary. But it _is_ a little scary, or has the potential to be. Switching to warn